src/convertToObject.js

```diff
@@ -3,16 +3,38 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleanedUpString = sourceString
+    .split('\n')
+    .join('')
+    .trim()
+    .split(';')
+    .map(el => el.split(':'));
+
+  const result = {};
+
+  cleanedUpString.forEach(element => {
+    if (element.length > 1) {
+      const [cssProperty, cssValue] = element;
+
+      const objectKey = cssProperty.trim();
+      const objectValue = cssValue.trim();
+
+      result[objectKey] = objectValue;
```

Comment:
```suggestion
      const objectKey = cssProperty.trim();
      const objectValue = cssValue.trim();

      result[cssProperty.trim()] = cssValue.trim();
```

It would save you some coding just to implement it directly in the result